RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py

```diff
@@ -17,7 +17,25 @@
   TruncatePixelCharge = False,
   IrradiationBiasCorrection = False, # set IBC off
   DoCosmics = False,
-  Upgrade = True                     # use 'upgrade' version of hardcoded CPE errors
+  Upgrade = True,                    # use 'upgrade' version of hardcoded CPE errors
+  xerr_barrel_ln_ = cms.vdouble(0.00025, 0.00030, 0.00035, 0.00035),
```

Comment:
there is an extra underscore here and in all the errors below.
This leads to:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-87155f/15023/runTheMatrix-results/23234.0_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal/step3_TTbar_14TeV+2026D49+TTbar_14TeV_TuneCP5_GenSimHLBeamSpot14+DigiTrigger+RecoGlobal+HARVESTGlobal.log

Reply:
corrected